addons/event_sale/wizard/event_configurator.py

```diff
@@ -11,3 +12,35 @@ class EventConfigurator(models.TransientModel):
     product_id = fields.Many2one('product.product', string="Product", readonly=True)
     event_id = fields.Many2one('event.event', string="Event")
     event_ticket_id = fields.Many2one('event.event.ticket', string="Event Ticket")
+    has_available_events = fields.Boolean(
+        "Has Available Events",
+        compute="_compute_has_available_events",
+        help="Technical field used to know if the user has at least one event that can be selected and used to configure event tickets on sale order lines")
+
+    @api.model
+    def default_get(self, fields):
+        res = super(EventConfigurator, self).default_get(fields)
+        if 'event_id' in fields and res.get('product_id'):
+            upcoming_events = self.env['event.event'].search(
+                [('event_ticket_ids.product_id', '=', res['product_id']),
+                 ('date_end', '>=',
```

Comment:
Could be great to have a computed / searchable field for that,  a bit like is_ongoing or is_one_day . I saw a PR with that field one day, not sure where / when / who ^^ ...